docs/clang-tidy.md

```diff
@@ -30,11 +30,11 @@ native_binary(
 Finally, create the linter aspect, typically in `tools/lint/linters.bzl`:
 
 ```starlark
-load("@aspect_rules_lint//lint:clang_tidy.bzl", "clang_tidy_aspect")
+load("@aspect_rules_lint//lint:clang_tidy.bzl", "lint_clang_tidy_aspect")
```

Comment:
this file is generated from clang-tidy.bzl - could you make a matching update there to get tests green?